src/arrays.js

```diff
@@ -1,61 +1,133 @@
 const getNthElement = (index, array) => {
-  // your code here
+  if (index >= array.length){
+  return array[index - array.length];
+  } else{
+  return array[index];
+  }
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  return array.toString();
 };
 
 const csvStringToArray = string => {
-  // your code here
+  return string.split(',');
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  const array2 = array.concat(element);
```

Comment:
You could return this straight away without assigning to a const. Like `return array.concat(element);`